### DISCUSSION TITLE
Folders colors - Why?

### DISCUSSION THREAD

**Discussion by LaptosPirose** [atom/atom#22545] [Q&A]
Hello! I'm just beginning in ATOM and I notice that some folders in my project tree has different colors. Anyone know what the colors means? Where can I find some documentations about ATOM, in order to learn more?

I don't have any expert packages installed. Just Git, Prettier, Live Server and Tabnine.
<img width="212" alt="Captura de Tela 2021-06-05 às 07 54 57" src="https://user-images.githubusercontent.com/84459248/120889421-5765e900-c5d3-11eb-9461-98263ce9d728.png">

**Comment by ThatXliner** [comment]
Yellow means modified. Green means added. This is a git repo perk
